src/main/webapp/index-script.js

```diff
@@ -44,51 +69,62 @@ function loadMainButtons() {
     var zipcode = document.getElementById('searchForm').elements[1].value;
     searchByText(query, zipcode);
   });
-  
+
+  // Prevent page from refreshing when you submit the form
+  searchForm.addEventListener('submit', function(event) {
```

Comment:
I'm confused why you keep moving this code.

Reply:
The first move was to decompose the loadMainButtons() function since there was a ton of code inside of that one function. Any other movements must have been me trying to fix resulting merge conflicts.